README.md

```diff
@@ -1 +1,128 @@
 # central-uploader
+
+The `central-uploader` tool is to enable application release artifacts to Launchpad project pages.
+
+
+# How it works
+
+In order to be able to use the `central-uploader`, we need a Launchpad project, with a build proces,
+producing artifacts as described in the **Project Built from Source (prerequisite)** section below.
+
+The `centra-uploader`'s task in the chain is to download the existing artifacts from the
+strictly protected Launchpad SOSS space, and publish them to the project's public Launchpad releases space.
+
+## Registering a project
+
+If you want to register your project with the `central-uploader`, you need to add a section
+as below to the `products.json` file, and run the pipelines.
```

Comment:
Maybe it is worth saying that rather than "you need to add a section" I would say that people should "raise a PR" :smile: